tests/integration_tests/test_server.py

```diff
@@ -238,6 +232,103 @@
         await asyncio.wait_for(read_until_command(proto3, "game_info"), 0.2)
 
 
+@fast_forward(300)
+async def test_game_info_broadcast_on_connection_error(
+    event_loop, lobby_server, tmp_user, ladder_service, game_service, caplog
+):
+    """
+    Causes connection errors in `do_report_dirties` which in turn will cause
+    closed games not to be cleaned up if the errors aren't handled properly.
+    """
+    # This test causes way to much logging output otherwise
+    caplog.set_level(logging.WARNING)
+
+    NUM_HOSTS = 10
+    NUM_PLAYERS_DC = 20
+    NUM_TIMES_DC = 10
```

Comment:
`NUM_PLAYERS_DC` + `NUM_TIMES_DC` what that `DC` is?

Reply:
Disconnect

Reply:
Notice this line 
```python
# Set up our players that will disconnect
dc_players = [await tmp_user("Disconnecter") for _ in range(NUM_PLAYERS_DC)]
```